packages/deployer/terraform/airnode/main.tf

```diff
@@ -0,0 +1,65 @@
+# TODO:
+# * switch between local and remote lambda source
+# * variable validation
+
+module "initializeProvider" {
+  source = "./modules/function"
+
+  name                = "initializeProvider"
+  handler             = "handlers/aws/index.initializeProvider"
+  source_file         = "${path.module}/../../.webpack/handlers/aws/index.js"
+  timeout             = 20
+  infrastructure_name = var.infrastructure_name
+  stage               = var.stage
+  airnode_id_short    = var.airnode_id_short
+  configuration_file  = var.configuration_file
+  secrets_file        = var.secrets_file
+}
+
+module "callApi" {
+  source = "./modules/function"
+
+  name                = "callApi"
+  handler             = "handlers/aws/index.callApi"
+  source_file         = "${path.module}/../../.webpack/handlers/aws/index.js"
+  timeout             = 30
+  infrastructure_name = var.infrastructure_name
```

Comment:
You can do somthing like this https://stackoverflow.com/a/56449409
Basically instead of passing multiple different variables you're passing one object variable. But I don't think it makes much sense here. I would use something like that only if the variables have some connection between each other, e.g. metadata variables or some such. So, yeah, I don't think it's worth it.

Reply:
Yeah I thought as much 👍🏻